diff --git a/node_modules/typewriter-effect/index.d.ts b/node_modules/typewriter-effect/index.d.ts
index 3541220..623b05f 100644
--- a/node_modules/typewriter-effect/index.d.ts
+++ b/node_modules/typewriter-effect/index.d.ts
@@ -39,6 +39,12 @@ declare module "typewriter-effect" {
      * @default false
      */
     autoStart?: boolean
+    /**
+     * The pause duration after a string is typed when using autostart mode.
+     * 
+     * @default 1500
+     */
+    pauseFor?: number
     /**
      * Whether or not to display console logs.
      *
